<launch>

  <arg name="x" default="0"/>
  <arg name="y" default="0"/>
  <arg name="robot_ns" default="hydrus" />

  <node name="hydrus_position_reset" type="hydrus_position_reset.py" pkg="mbzirc2020_task2_simulation" ns="$(arg robot_ns)" output="screen">
    <param name="respawn_pos_x" value="$(arg x)"/>
    <param name="respawn_pos_y" value="$(arg y)"/>
    <param name="tf_prefix" value="$(arg robot_ns)"/>
  </node>

</launch>
